src/Types/VersionedModuleSource.cs

```diff
@@ -1,11 +1,10 @@
+using System.Buffers.Binary;
 using Concordium.Sdk.Exceptions;
+using Concordium.Sdk.Helpers;
 using WebAssembly;
 
 namespace Concordium.Sdk.Types;
 
-/// <summary>
-/// Contains source code of a versioned module where inherited classes are concrete versions.
```

Comment:
Was this header wrong?

Reply:
No this is definitely a merge error, my bad.